components/common/header/logo.js

```diff
@@ -9,13 +10,15 @@ Even that short amount of time for the useMedia hook to run was long enough to c
 */
 
 const FBLogo = ({ authed }) => (
-  <Link href={authed ? '/dashboard' : '/'} aria-label='Flossbank – Home'>
-    <Box display={{ base: 'none', md: 'initial' }} aria-hidden='true'>
-      <FullLogo />
-    </Box>
-    <Box display={{ base: 'initial', md: 'none' }} aria-hidden='true'>
-      <FBLogoLetters id='letter-logo-header' />
-    </Box>
+  <Link href={authed ? '/dashboard' : '/'}>
+    <ChakraLink aria-label='Flossbank – Home'>
+      <Box display={{ base: 'none', md: 'initial' }} aria-hidden='true'>
+        <FullLogo />
+      </Box>
+      <Box display={{ base: 'initial', md: 'none' }} aria-hidden='true'>
+        <FBLogoLetters id='letter-logo-header' />
+      </Box>
+    </ChakraLink>
```

Comment:
I'm confused as to why we need `ChakraLink` here at all instead of just using `Link`? We're not styling the actual link so we don't need what Chakra is offering here. Swapping Link to next/link and leaving the component code as it was should work.

Reply:
this was done for a few reasons. we need an <a> tag within the next/Link so that it's accessible as a "link" (do i have that right?), but also next link can't take in more than one child element so this was also to wrap the children. i could have put just a plain <a> but since ChakraLink is the same thing and allows for extensibility i used that instead

Reply:
going to merge for now to avoid merge conflicts in a PR today! let's shelve this convo